clients/swift/Package.swift

```diff
@@ -0,0 +1,30 @@
+// swift-tools-version: 5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Readiness",
+    platforms: [.iOS(.v15), .macOS(.v10_15)],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        .package(url: "https://github.com/jpsim/Yams.git", from: "5.0.6"),
+//        .package(url: "https://github.com/polywrap/swift-client", branch: "main")
+        .package(path: "../../../swift/PolywrapClient")
```

Comment:
How does this work in CI? I'm confused as to how it's able to install this. Can we change this back to the github URL?

Reply:
Ideally we'd be installing the Swift client as if we were a 3rd party developer application.

Reply:
lol nice catch this was for development purposes